docs/README.md

```diff
@@ -0,0 +1,18 @@
+# Documentation
+
+This documentation is using the [Sphinx generator](https://www.sphinx-doc.org/en/master/) to create the whole documentation and are building automatically using the GiHub Actions.
+
+The custom theme is the official [Qiskit theme for Ecosystem project](https://qiskit.github.io/qiskit_sphinx_theme/), do not hesitate to go check the documentation in order to know how to custimize it further.
+
+The documentation is structured following the [Diátaxis Framework](https://diataxis.fr/).
+This framework is for technical documentation that divides it into 4 categories according to the users needs.
```

Comment:
Nice! This is great. I recommend though that you instead link to https://qiskit.github.io/qiskit_sphinx_theme/ which explains this more fully about Diataxis and has several examples and best practices. It will help to de-duplicate the content.

Reply:
Something like that : https://github.com/qiskit-community/quantum-prototype-template/pull/72/commits/d62db836a64f3471bf508f6be485cf0226df81cb ?